HISTORY.rst

```diff
@@ -12,7 +24,6 @@ Next Release
 0.8.5 (2018-08-20)
 ------------------
 * Unpin cobra dependency and set it to >0.13.3.
-* Set ruamel.yaml to >=0.15 to keep up with cobra.
```

Comment:
Did you rebase your branch on the latest devel? I'm confused why this line should be removed?